verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -131,11 +131,11 @@ export default class LinelistTable extends React.Component<{}, LinelistTableStat
             if (!this.validateRequired(newRowData.source_url)) {
                 return reject();
             }
+            console.log('here');
```

Comment:
nit: remove (& below)

this is my MO, i get it

Reply:
:P 
done